frontend/src/lib/layout/AppMenu.svelte

```diff
@@ -11,8 +11,8 @@
   <!-- https://daisyui.com/components/menu  -->
   <ul class="menu bg-base-100 min-w-[33%]">
     <header class="prose flex flex-col items-end p-4 mb-4">
-      <h2 class="mb-0">{$user?.name}</h2>
-      <span class="font-light">{$user?.email}</span>
+      <h2 class="mb-0">{$page.data.user?.name}</h2>
```

Comment:
Ultimately we could create a user store that derives from `$page` right?
We can configure lint to forbid certain imports in certain places. I'm looking forward to have a good reason to make use of that feature 😆

Reply:
We could, but I don't think it's a good idea. `$page.data.user` is type-safe, and doesn't break the Law of Demeter because `$page.data` is guaranteed by the Svelte docs to always exist. So `user = derived(page, p => p.data.user)` is just unnecessary complexity IMHO.